app/services/resolve_account_service.rb

```diff
@@ -106,8 +106,6 @@ def split_acct(acct)
   end
 
   def fetch_account!
-    return unless activitypub_ready?
```

Comment:
Just to be sure: You removed this because `Webfinger#validate_response!` will now raise if that is not the case. Right?

Reply:
Brilliant, thanks!